internal/cmd/shell/shell.go

```diff
@@ -54,7 +54,9 @@ second argument:
 			database := args[0]
 
 			if !printer.IsTTY || ch.Printer.Format() != printer.Human {
-				return errors.New("pscale shell only works in interactive mode")
+				if !(os.Getenv("PSCALE_ALLOW_NONINTERACTIVE_SHELL") == "true") {
```

Comment:
I think we should just check whether it exists or not, instead of checking the value:

```suggestion
				if os.Getenv("PSCALE_ALLOW_NONINTERACTIVE_SHELL") == "" {
```